Composer/packages/extensions/adaptive-flow/package.json

```diff
@@ -28,6 +28,8 @@
     "@bfc/ui-shared": "*",
     "@emotion/core": "^10.0.27",
     "@emotion/styled": "^10.0.27",
+    "adaptive-expressions": "4.10.0-preview-147186",
+    "botbuilder-lg": "^4.10.0-preview-150886",
```

Comment:
"botbuilder-lg" already contains the corresponding "adaptive-expressions" library

Reply:
Are expression utils exported from the botbuilder-lg pkg as well? I didn't find them

Reply:
As a discussion result, we only use the `adaptive-expressions` package now. mark as resovled